services/ui-src/src/components/forms/AdminBannerForm.tsx

```diff
@@ -33,9 +37,11 @@ export const AdminBannerForm = ({ writeAdminBanner, ...props }: Props) => {
     };
     try {
       await writeAdminBanner(newBannerData);
+      setLoading(false);
       window.scrollTo(0, 0);
     } catch (error: any) {
       setError(bannerErrors.REPLACE_BANNER_FAILED);
+      setLoading(false);
```

Comment:
if we want to set this to false in the try and catch block, we could just put it at the end of the method